Collections/ArrayListHashSet.java

```diff
@@ -0,0 +1,18 @@
+import java.util.*;
+
+public class ArrayListHashSet {
+    public static void toHashSet() {
+
+        ArrayList<String> fruits = new ArrayList<>();
+        fruits.addAll(Arrays.asList("Apple", "Pear", "Banana", "Apple", "Cherry", "Fig", "Orange", "Banana", "Apple"));

```

Comment:
You can also do it like this:
`ArrayList<String> fruits = List.of("Apple", "Pear", "Banana", "Apple", "Cherry", "Fig", "Orange", "Banana", "Apple")`